### DISCUSSION TITLE
2FA w/o email?

### DISCUSSION THREAD

**Discussion by smatovic** [community/community#65067] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

W/o 2FA per email github will be unusable for me and I will move with my projects to another host, or host by myself.

**Comment by jmcunx** [comment]
Yes, that is my plan, I already copied to anonymous ftp, but looking at gitlab.  But I am hoping thet will allow email

https://docs.gitlab.com/ee/user/project/import/github.html
